### PR TITLE
Add animated texture atlases

### PR DESCRIPTION
**Pull request by Rayoule**
New way to handle texture atlases with a new AnimatedIndex that allows to animate the texture atlas index !
New example to show how it works  with the same atlas as [bevy's example](https://bevyengine.org/examples/2d/sprite-sheet/) mentioned in the issue #31 
I did not exactly applied the same method as mentioned, but don't hesitate to tell me how you find that solution !

### CODE REVIEW

File: assets/gabe-idle-run.png
Comment (abnormalbrain) [inline assets/gabe-idle-run.png:1]: May I ask where you got this image? Mostly, I want to verify that including it in the repo isn't a licensing issue.

**Review by abnormalbrain — COMMENTED** [review]
This looks great!

File: assets/gabe-idle-run.png
Comment (Rayoule) [reply assets/gabe-idle-run.png:1]: Yes I took it from the bevy [repo](https://github.com/bevyengine/bevy/tree/main/assets/textures/rpg/chars/gabe)
I put the link in the example. Should we add something ?

Comment (abnormalbrain) [reply assets/gabe-idle-run.png:1]: Ah gotcha. Nope, that works, I think.

**Pull request merged**